src/Perpetuum/Modules/SlotFlags.cs

```diff
@@ -1,20 +1,24 @@
-﻿namespace Perpetuum.Modules
+﻿using System;
+
+namespace Perpetuum.Modules
 {
+    /// Doc - These are powers of 8
```

Comment:
powers of 2?

Reply:
Yep, I was drinking something that night apparently.